test/win32/test_registry.rb

```diff
@@ -39,9 +52,9 @@ def test_predefined
     end
 
     def test_open_no_block
-      Win32::Registry::HKEY_CURRENT_USER.create(backslachs(TEST_REGISTRY_KEY)).close
+      Win32::Registry::HKEY_CURRENT_USER.create(@test_registry_key).close
 
-      reg = Win32::Registry::HKEY_CURRENT_USER.open(backslachs(TEST_REGISTRY_KEY), Win32::Registry::KEY_ALL_ACCESS)
+      reg = Win32::Registry::HKEY_CURRENT_USER.open(@test_registry_key, Win32::Registry::KEY_ALL_ACCESS)
```

Comment:
Since registry keys can include '/', it is wrong to use `File` to separate them.

Reply:
It's only used for testing, where we have full control about our test keys. Nevertheless I'll change it.

Reply:
My concern is that it might encourage to write wrong code in other places.